### ISSUE TITLE
Pulling and filing email is having trouble

### ISSUE THREAD

**Issue by ghost** [ABTech/tracker#76]
Firstoff, there are now two copies of all recent emails in the tracker email section.  That's an annoying issue.  

Second, whenever I pull from the inbox, it marks all the messages in the inbox as flagged.  

Third, it appears that it pulls every single message no matter what--when I go to file emails, it's starting me back in 2005.  

Fourth, it either ignores the mark as ignored button and skip to next button sometimes, or there are so many copies it takes like 10 clicks to get to the next email (I can't tell because I can't skip to the last page of emails in the tracker).  

Fifth, it takes a stupidly long time to file an email.  I'm not sure if this is a slow server thing or another thing, but if the server needs more RAM or something, let me know.  

Basically, the whole email thing is nonfunctional right now, though I'm not sure exactly why or how, but that's a big issue.
